blog/2022-11-09-apache-monitoring.md

```diff
@@ -0,0 +1,144 @@
+
+![image](https://user-images.githubusercontent.com/24860547/200813567-ae74d0e4-b8e6-443a-930e-49ddb2e11851.png)
+
+The Apache Scoreboard is a very useful chart if you are trying to troubleshoot issues with your web server. The traditional view of the scoreboard looks like this:
+
+![image](https://user-images.githubusercontent.com/24860547/200816766-29a42d42-4979-4d76-b499-be6cc2dfbd09.png)
+
+In Netdata, the scoreboard is represented as a helpful chart and you don't need to worry about remembering all of the acronyms. 
+
+![image](https://user-images.githubusercontent.com/24860547/200813628-53486c70-87d9-43ae-a690-6d3b3e473a02.png)
+
+- A scoreboard that displays a large amount of `sending` may indicate a poorly performing web application such as a PHP website. Combined with a large number of traffic spikes, it could be indicative of a DoS attack. 
+
+- A large amount of `reading` on the other hand may indicate a [Slowloris](https://en.wikipedia.org/wiki/Slowloris_(computer_security)) attack, where many connections are opened and kept open for as long as possible.
+
+- A lot of connections in a keep-alive state, may indicate that the server is getting many requests from clients that do not make subsequent requests (and therefore do not help you reap the intended benefits of keep-alive connections). 
+
+### Bandwidth
+
+The bandwidth handled by the Apache server (measured in megabits per second) is another important metric that helps you understand the [load](https://blog.netdata.cloud/server-load#what-is-server-load) your server is currently handling.
+
+![image](https://user-images.githubusercontent.com/24860547/200813688-67011d91-2fd1-48ee-a492-bb7409609e06.png)
+
+### Workers
+
+Worker resources are important to monitor as this tells you which resources are over and under utilized. 
+
+The worker threads chart represents worker utilization. 
+
+A worker thread that is in any of the following states is considered busy: 
+- reading
+- writing
+- keep-alive
+- logging
+- closing
+- gracefully finishing
+
+A worker thread not in any of the states mentioned above is considered idle. 
+
+A consistently large number of idle workers (as seen in the example chart here) indicates that more threads are in use than are necessary for the current traffic levels and load. This will lead to unnecessary utilization of system resources and you may consider lowering the `MinSpareThreads` configuration parameter.
+
+If however you only have a very small number of idle workers consistently this could lead to slowing down your server and requests getting queued up when the `MaxRequestWorkers` limit is hit. Increasing the `MaxRequestWorkers` can help with this scenario but be mindful that each extra worker thread requires extra system resources. 
```

Comment:
```suggestion
If however you only have a very small number of idle workers consistently this could lead to slowing down your server and requests getting queued up when the [`MaxRequestWorkers`](https://httpd.apache.org/docs/2.4/mod/mpm_common.html#maxrequestworkers) limit is hit. Increasing the [`MaxRequestWorkers`](https://httpd.apache.org/docs/2.4/mod/mpm_common.html#maxrequestworkers) can help with this scenario but be mindful that each extra worker thread requires extra system resources. 
```